### DISCUSSION TITLE
No Keyboard events when connecting from Windows 11 to Windows 11.

### DISCUSSION THREAD

**Discussion by pjd** [FreeRDP/FreeRDP#9755] [General]
Hi. I have a proxy between two Windows 11 systems. For any other combination I can see Keyboard and Mouse events as in RDP protocol documentation, but when Windows 11 connects to another Windows 11 the keyboard events are sent within drdynvc channel. I'm trying to figure out how is this negotiated. Can someone point me at the right direction? Thank you.

**Comment by akallabeth** [comment]
[edit] after rereading your question, you might want to check out [[MS-RDPECI]](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpeci/7be99a27-1c39-47f1-a76a-e38869ee892a)

**Comment by pookaboydunc** [comment]
@pjd did you ever get this working again? I have a similar issue.
